packages/flame_svg/lib/svg.dart

```diff
@@ -55,11 +55,24 @@ class Svg {
     Vector2 size, {
     Paint? overridePaint,
   }) {
-    final localSize = size.toSize();
+    // Scale the canvas to the size of the destination clip bounds
+    // This is necessary to avoid blurriness when having a
+    // camera.viewfinder.zoom larger than 1.0
+    final widthRatio = canvas.getDestinationClipBounds().size.width /
+        canvas.getLocalClipBounds().size.width;
+    final heightRatio = canvas.getDestinationClipBounds().size.height /
+        canvas.getLocalClipBounds().size.height;
+
+    final localSize = Size(size.x * widthRatio, size.y * heightRatio);
     final image = _getImage(localSize);
 
     canvas.save();
-    canvas.scale(1 / pixelRatio);
+    canvas.scaleVector(
```

Comment:
Since we don't have a `Vector2` already we should avoid creating a new one here, just use the normal `canvas.scale(x, y)`.

Reply:
Applied in 7d0dcb46673f37dda87fc80ae27ea1ba837ef57f